hs_access_control/models/user.py

```diff
@@ -208,6 +209,46 @@ def act_on_group_membership_request(self, this_request, accept_request=True):
         this_request.redeemed = True
         this_request.save()
 
+    def redeem_group_membership_request(self, this_request, for_user=None):
```

Comment:
I don't think this method is needed?  It seems to be mostly a duplicate of act_on_group_membership_request above.  The for_user parameter is checked and assigned after the check but never used.

Reply:
fixed this thanks Scott